src/WebJobs.Script.Grpc/Abstractions/IRpcServer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public interface IRpcServer
```

Comment:
For all the new public types you're adding in this library, please add full xml documentation, since my understanding is that this library will be used by language implementors.

Reply:
Also, the naming of this project is specific to Grpc, but your interface here is generalized to Rpc? Is that because Grpc is just one potential specialization? If so, should the project be generalized to WebJobs.Script.Rpc?

Reply:
See above - need to get general abstractions project that is separate from the Grpc implementation.

Reply:
Good comment... see my suggestion above.